mne/datasets/config.py

```diff
@@ -0,0 +1,217 @@
+   (available through a separate process) might allow identification of
+   individuals.  Different committees operate under different national, state
+   and local laws and may interpret regulations differently, so it is
+   important to ask about this. If needed and upon request, the HCP will
+   provide a certificate stating that you have accepted the HCP Open Access
+   Data Use Terms.
+
+4. I may redistribute original WU-Minn HCP Open Access data and any derived
+   data as long as the data are redistributed under these same Data Use Terms.
+
+5. I will acknowledge the use of WU-Minn HCP data and data derived from
+   WU-Minn HCP data when publicly presenting any results or algorithms
+   that benefitted from their use.
+
+   1. Papers, book chapters, books, posters, oral presentations, and all
+      other printed and digital presentations of results derived from HCP
+      data should contain the following wording in the acknowledgments
+      section: "Data were provided [in part] by the Human Connectome
+      Project, WU-Minn Consortium (Principal Investigators: David Van Essen
+      and Kamil Ugurbil; 1U54MH091657) funded by the 16 NIH Institutes and
+      Centers that support the NIH Blueprint for Neuroscience Research; and
+      by the McDonnell Center for Systems Neuroscience at Washington
+      University."
+
+   2. Authors of publications or presentations using WU-Minn HCP data
+      should cite relevant publications describing the methods used by the
+      HCP to acquire and process the data. The specific publications that
+      are appropriate to cite in any given study will depend on what HCP
+      data were used and for what purposes. An annotated and appropriately
+      up-to-date list of publications that may warrant consideration is
+      available at http://www.humanconnectome.org/about/acknowledgehcp.html
+
+   3. The WU-Minn HCP Consortium as a whole should not be included as an
+      author of publications or presentations if this authorship would be
+      based solely on the use of WU-Minn HCP data.
+
+6. Failure to abide by these guidelines will result in termination of my
+   privileges to access WU-Minn HCP data.
+"""
+
+# To update the `testing` or `misc` datasets, push or merge commits to their
+# respective repos, and make a new release of the dataset on GitHub. Then
+# update the checksum in `mne/data/dataset_checksums.txt` and change version
+# here:                  ↓↓↓↓↓         ↓↓↓
+RELEASES = dict(testing='0.112', misc='0.8')
+
+# To update any other dataset besides `testing` or `misc`, upload the new
+# version of the data archive itself (e.g., to https://osf.io or wherever) and
+# then update the corresponding checksum in `mne/data/dataset_checksums.txt`.
+TESTING_VERSIONED = f'mne-testing-data-{RELEASES["testing"]}'
+MISC_VERSIONED = f'mne-misc-data-{RELEASES["misc"]}'
+
+# remote locations of the various datasets
+URLS = dict(
```

Comment:
might make sense to alphabetize these lines by key, just to make future devs a little happier.